site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -462,14 +539,26 @@ def _transform_subjects(self):
         """Mapping of keywords to subjects."""
         keywords = self.inspire_metadata.get("keywords", [])
         mapped_subjects = []
+
         for keyword in keywords:
             value = keyword.get("value")
-            if value:
-                mapped_subjects.append(
-                    {
-                        "subject": value,
-                    }
-                )
+            schema = keyword.get("schema", "").upper()
+
+            if not value:
+                continue
+
+            if schema in ["PACS", "CERN LIBRARY"]:
```

Comment:
lets maybe leave a comment that it was agreed to drop these schemes.
Did you see maybe UDC schema in any of the records?